dist/index.html

```diff
@@ -3,17 +3,86 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <!-- font awesome file -->
+    <link rel="stylesheet" href="./assets/css/font-awesome.min.css" />
+
     <link rel="stylesheet" href="./assets/css/main.css" />
     <title>Charity</title>
   </head>
   <body>
     <header>
-      <section class="contact">
-        Welcome text and social media Links
+      <section class="contact bg-c100 text-c700 py-2">
+        <div class="container flex justify-between">
+          <div class="welcome-text text-sm">
+            Welcome to the best
+            <span class="text-c300 underline italic">Lovims</span> charity
+            platform
+          </div>
+          <div class="social flex text-sm">
+            <div>Follow us:</div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-twitter" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-facebook-square" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-pinterest-p" aria-hidden="true"></i>
+              </a>
+            </div>
+            <div class="pl-6 hover:text-c000">
+              <a href="#">
+                <i class="fa fa-instagram" aria-hidden="true"></i>
+              </a>
+            </div>
+          </div>
+        </div>
       </section>
 
-      <section>
-        Logo And Contact Info
+      <section class="logo-section py-8">
+        <div class="container py-2 flex">
+          <div class="logo w-1/2">
+            <img src="assets/Logo@1X.png" alt="" />
```

Comment:
Why we need to add alt for every image.
The below definition from MDN

> The HTMLImageElement property alt provides fallback (alternate) text to display when the image specified by the <img> element is not loaded. This may be the case because of an error, because the user has disabled the loading of images, or because the image simply hasn't finished loading yet.

You can read more in this article too [writing effective alt for imagest](https://inviqa.com/blog/writing-effective-alt-text-images)